src/Persistence.Tests/Yaml/DeserializerInvalidTests.cs

```diff
@@ -21,31 +21,13 @@ public void Deserialize_ShouldFailWhenYamlIsInvalid(bool isControlIdentifiers)
         // Uncomment to test single file
         // var files = new string[] { @"_TestData/InvalidYaml/Screen-with-host.pa.yaml" };
 
-        var failedFiles = 0;
-        Parallel.ForEach(files, file =>
+        foreach (var filePath in files)
         {
-            using var yamlStream = File.OpenRead(file);
```

Comment:
I am curious why you are removing parallel and making tests sequential.

Reply:
parallel execution is not needed for this test. parallelize at the top layer instead. the new test code is simpler and more explicit about what's being expected than previous.

note, I only changed this here because of a couple rule failures. Turns out this test wasn't very good at describing the expectation of an exception being expected either.

Reply:
this unit test will be updated when I implement the swallowing of all yamldotnet exceptions too

Reply:
* I mean, parallelize at the test runner level, not within a test.
*